src/hugr/views/tests.rs

```diff
@@ -38,7 +51,19 @@ fn node_connections() -> Result<(), BuildError> {
                 PortOffset::new_outgoing(1).into(),
                 PortOffset::new_incoming(0).into()
             ],
+            // order edge
+            [
+                PortOffset::new_outgoing(2).into(),
+                PortOffset::new_incoming(2).into()
+            ],
         ]
     );
     Ok(())
 }
+
+#[rstest]
+fn dot_string(sample_hugr: (Hugr, BuildHandle<DataflowOpID>, BuildHandle<DataflowOpID>)) {
+    let (h, _, _) = sample_hugr;
+
+    insta::assert_yaml_snapshot!(h.dot_string());
```

Comment:
This is not yaml, so probably should be `assert_snapshot`? Or is it needed for whitespace stuff?

Reply:
it's used for easier debug, its the snap format